fedorable.sh

```
@@ -1,334 +1,197 @@
-# Requires root privileges. Use with caution. Review --help output.
+# Performs system updates, cleanup, and optimizations
+# Requires root privileges. Use with caution.
 
 # Strict error handling
 set -euo pipefail
 
-# --- Configuration (Defaults - Can be overridden by /etc/fedorable.conf) ---
+#=============================================
+# CONFIGURATION
+#=============================================
 readonly SCRIPT_NAME=$(basename "$0")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)